articles/user-guide/language/expressions.md

```diff
@@ -0,0 +1,558 @@
+﻿---
+title: Type Expressions in Q#
+description: Understand how to specify, reference and combine constants, variables, operators, operations and functions as expressions in Q#.
+author: gillenhaalb
+ms.author: a-gibec@microsoft.com
+ms.date: 03/05/2020
+ms.topic: article
+uid: microsoft.quantum.guide.expressions
+---
+
+# Type Expressions in Q#
+
+## Numeric Expressions
+
+Numeric expressions are expressions of type `Int`, `BigInt`, or `Double`.
+That is, they are either integer or floating-point numbers.
+
+`Int` literals in Q# are identical to integer literals in C#,
```

Comment:
Would it make sense to make this more explicit for readers not used to C# (e.g.: Python users)?